src/main/cpp/t10_unique.cpp

```diff
@@ -26,4 +26,26 @@ using namespace std;
 
 int t10_unique() {
 
+    int n;
+    cin >> n;
+    int arr[n];
+    int a;
+    bool shouldCout;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'shouldCout' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4621039)